### PR TITLE
Add a fallback command to retry reinstalling guest agent during startup/shutdown script testing

### PR DESCRIPTION
**Pull request by a-crate**
dnf and yum don't have a command that forces a reinstall or an upgrade
Also swapped the argument order for zypper, newer version require the --force flag after the install command
/cc @koln67 @drewhli

### CODE REVIEW

File: imagetest/test_suites/metadata/startup_script_test.go [hunk 1]
```diff
@@ -103,21 +103,29 @@ func TestStartupScripts(t *testing.T) {
...
 		}
 	} else {
-		var cmd *exec.Cmd
+		var cmd,fallback *exec.Cmd
 		switch {
 		case utils.CheckLinuxCmdExists("apt"):
 			cmd = exec.Command("apt", "reinstall", "-y", "google-guest-agent")
 		case utils.CheckLinuxCmdExists("dnf"):
 			cmd = exec.Command("dnf", "-y", "reinstall", "google-guest-agent")
+			cmd = exec.Command("dnf", "-y", "upgrade", "google-guest-agent")
```
Comment (koln67) [inline imagetest/test_suites/metadata/startup_script_test.go:112]: minor typo: the second line probably should be fallback = exec.Command("dnf", "-y", "upgrade", "google-guest-agent")

Comment (a-crate) [reply imagetest/test_suites/metadata/startup_script_test.go]: Woops, yeah.

**Pull request merged**